stats/view/aggregation_data.go

```diff
@@ -89,18 +92,35 @@ func (a *SumData) equal(other AggregationData) bool {
 //
 // Most users won't directly access distribution data.
 type DistributionData struct {
-	Count           int64     // number of data points aggregated
-	Min             float64   // minimum value in the distribution
-	Max             float64   // max value in the distribution
-	Mean            float64   // mean of the distribution
-	SumOfSquaredDev float64   // sum of the squared deviation from the mean
-	CountPerBucket  []int64   // number of occurrences per bucket
-	bounds          []float64 // histogram distribution of the values
+	Count           int64   // number of data points aggregated
+	Min             float64 // minimum value in the distribution
+	Max             float64 // max value in the distribution
+	Mean            float64 // mean of the distribution
+	SumOfSquaredDev float64 // sum of the squared deviation from the mean
+	// Deprecated: Replaced by Count in Buckets.
+	CountPerBucket []int64   // number of occurrences per bucket
+	bounds         []float64 // histogram distribution of the values
+	Buckets        []Bucket
+}
+
+// ExemplarData is an example data point associated with each bucket.
+type ExemplarData struct {
+	Value       float64           // the value that was recorded
```

Comment:
Nice! I was actually thinking to comment about making this a float64, but you already did it.